airflow/models/taskinstance.py

```diff
@@ -1079,6 +1080,8 @@ def signal_handler(signum, frame):
         except (Exception, KeyboardInterrupt) as e:
             self.handle_failure(e, test_mode, context)
             raise
+        finally:
+            Stats.incr('ti.finish.{}.{}.{}'.format(task.dag_id, task.task_id, self.state), 1, 1)
```

Comment:
```suggestion
            Stats.incr('ti.finish.{}.{}.{}'.format(task.dag_id, task.task_id, self.state))
```

(I don't know why we include those two default params elsewhere.)

Reply:
Thank you @ashb can you make the same recommendation for the start task so I can just merged in the suggestion here too